content/git/concepts/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'cherry-pick'
+Description: 'The cherry pick command is used to select and put individual commits from one branch to another.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+The **cherry-pick** command entails picking individual [commits](https://www.codecademy.com/resources/docs/git/commit) from one [branch](https://www.codecademy.com/resources/docs/git/branch) to another.
+
+## Syntax
+
+```pseudo
+git cherry-pick <commit>
+```
```

Comment:
Add an explanation that `<commit>` is the hash and that it can be found using `git log`. Link the `git log` entry (https://www.codecademy.com/resources/docs/git/log) too.